coalib/bearlib/aspects/Smell.py

```diff
@@ -358,21 +358,44 @@ class docs:
 @Smell.subaspect
 class Complexity:
     """
+    This aspect checks on the complexity of your code.
```

Comment:
This is again an unrelated change. Refactoring one class should be separate from adding another.